src/lib/utils/uuid.js

```diff
@@ -0,0 +1,4 @@
+// polyfill Crypto.getRandomValues()
+import 'react-native-get-random-values'
+
+export { v4 as default } from 'uuid'
```

Comment:
this file is unnecessary
just import uuid as before

Reply:
@sirpy i started getting 'crypto.getRandomValues() not supported' while running android app locally.
so i had to add that polyfill